lib/cli-config.js

```diff
@@ -70,12 +72,16 @@ exports.getContent = function(config, directory) {
 
         if (ext === '.js' || ext === '.json') {
             content = require(configPath);
+        } else if (ext === '.yaml') {
+            content = yaml.safeLoad(fs.readFileSync(configPath, 'utf8'));
         } else {
-            content = JSON.parse(
-                stripJSONComments(
-                    fs.readFileSync(configPath, 'utf8')
-                )
-            );
+            data = fs.readFileSync(configPath, 'utf8');
+
+            try {
+                content = JSON.parse(stripJSONComments(data));
+            } catch (e) {
+                content = yaml.safeLoad(data);
```

Comment:
Yaml first would be more logical, but i think it is less clear, since other devs might not expect it and that might be considered as magic or as our own format